gaia/gdal_functions.py

```diff
@@ -0,0 +1,402 @@
+import string
```

Comment:
@mbertrand I am wondering if we can move this code to gaia/utils/geo but I am fine if we do it later as well. Also, can we not convert these into GaiaProcesses?